ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/dot1x.md

```diff
@@ -56,3 +56,11 @@ interface Management1
 | Service type | Framed MTU |
 | ------------ | ---------- |
 | True | 1500 |
+
+#### 802.1X Captive-portal Authentication
+
+| Authentication Attribute | Value |
+| ------------------------ | ----- |
+| URL | http://portal-nacm08/captiveredirect/ |
+| SSL profile | Profile1 |
+| Access-list IPv4 | ACL |
```

Comment:
missed supplicant logging ??

Reply:
supplicant logging is not inside `captive-portal` that's why didn't add it in this table.